tests/authentication/client/test_client_company.py

```diff
@@ -186,6 +186,37 @@ def test_get_client_company(self, mock_email):
                       str(response.data))
         self.assertEqual(response.status_code, status.HTTP_200_OK)
 
+    def test_get_clients_list_unauthenticated(self):
+        """Get clients."""
+        self.client.post(
+            self.client_url, self.valid_client_data, format="json")
+        response = self.client.get(
+            self.clients_list_url, format="json")
+        self.assertIn("Please log in to proceed.",
+                      str(response.data))
+
+    def test_get_clients_list(self):
+        """Get clients."""
```

Comment:
Hey @Rhytah, thanks for the good work on this PR, However, I realize you have the same docstring for `test_get_clients_list_unauthenticated` and `test_get_clients_list`, we could write more detailed docstrings to easily describe what each one does differently.

Reply:
could be something like 
```
    def test_get_clients_list_unauthenticated(self):
        """Get clients when user is unauthenticated"""
```